### PR TITLE
Run3-gex93B Modify Run3 and Phase2D86 geometry due to change in some material xml files (Backport #35290)

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Modify Run3 and Phase2D86 geometry due to change in some material xml files (Backport #35290)

#### PR validation:

Use the runTheMatrix test workflows

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Backport #35290 and #35210

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by cvuosalo** [comment]
+1

**Comment by srimanob** [comment]
@perrotta @qliphy 
Could you please confirm no change policy for production release in this case? Thanks.

**Comment by qliphy** [comment]
@srimanob   Run3 production has been launched with 12_0_X, and this PR indeed bring changes.
Adding @rappoccio who may comment more.
Or we can discuss this at next ORP.

**Comment by civanch** [comment]
We discussed at the recent SIM meeting if we should backport to 12_0 and agree that we prepare backports of all PRs related to geometry. This does not mean that all will be merged. We do this in order to be ready if a decision would be made to have 12_0 geometry equal to 12_1.

**Comment by srimanob** [comment]
Thanks @civanch 
I misunderstood that we need to merge it when PR is ready. So maybe I will hold it until we have a discussion at ORP.

**Comment by srimanob** [comment]
hold

**Comment by srimanob** [comment]
Following the release meeting, this PR can merge.

The 12_0 production uses 12_0_1 but with DB geometry equivalent to XML we have in 12_0_0. In principle, production can be controlled by GT. 

Note that, in the 12_0_X, autocond currently points to the new GT (with updated geometry) already
https://github.com/cms-sw/cmssw/blob/CMSSW_12_0_X/Configuration/AlCa/python/autoCond.py#L71
`120X_mcRun3_2021_realistic_v7` is advance one compared to `120X_mcRun3_2021_realistic_v5` (GS) and `120X_mcRun3_2021_realistic_v6` (DR) used in production. `v7` include updated in geometry introduced in 12_0_1.

**Comment by cvuosalo** [comment]
urgent

**Comment by perrotta** [comment]
backport of #35290

File: Configuration/Geometry/python/dict2021Geometry.py [hunk 1]
```diff
@@ -599,7 +599,7 @@
             'Geometry/TrackerCommonData/data/PhaseI/v2/pixfwdbladeInnerZminus.xml',
             'Geometry/TrackerCommonData/data/PhaseI/v2/pixfwdbladeOuterZplus.xml',
             'Geometry/TrackerCommonData/data/PhaseI/v2/pixfwdbladeOuterZminus.xml',
-            'Geometry/TrackerCommonData/data/PhaseI/v2/pixbarmaterial.xml', 
+            'Geometry/TrackerCommonData/data/PhaseI/v3/pixbarmaterial.xml',
```
Comment (perrotta) [inline Configuration/Geometry/python/dict2021Geometry.py:602]: Just trying to understand how this backport was designed: this line is from #34983, and it was apparently forgotten in some previous backport of that PR merged in master: could you please explain the logic and the history of previous backports?

Comment (bsunanda) [reply Configuration/Geometry/python/dict2021Geometry.py:602]: We had a few PR for 12_1_X and these are combined in the backport PR. Please merge this PR. It is in the way of other back-outings

**Comment by bsunanda** [comment]
@perrotta @qliphy Please merge

**Comment by qliphy** [comment]
> @perrotta @qliphy Please merge

I am fine with this PR.  I will let @perrotta have a check again and merge later if possible.

**Comment by perrotta** [comment]
@bsunanda, I'm trying to follow the complicate line of backports from several sources...
Why there is not the backport of the "v2" of tecmaterial.xml? Is it the same as "v1", and therefore it is notbackported for simplicity?

**Comment by bsunanda** [comment]
@perrotta We found later there is no need to have "v2" for tecmaterial. So a later PR (#35290) deletes "v2" for tecmaterial and current version in 12_1_X makes use of v1 version of tecmaterial.

**Comment by perrotta** [comment]
+1
- Backport of geometry updates that will not be picked up in the standard production workflows, for which no changes are expected

**Pull request merged**